spec/components/email_updater_spec.rb

```diff
@@ -17,6 +17,92 @@
     expect(updater.errors.messages[:base].first).to be I18n.t("change_email.error_staged")
   end
 
+  context "when an admin is changing the email of a user" do
+    let(:email_change_requesting_user) { Fabricate(:admin) }
```

Comment:
Calling this `admin` makes it easier to read

```suggestion
    let(:admin) { Fabricate(:admin) }
```